eras/shelley/impl/cardano-ledger-shelley.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-shelley
-version:            1.8.0.0
+version:            1.9.0.0
```

Comment:
Changelog needs to be updated with the new version and the breaking change

Reply:
done